pkgs/by-name/as/ascii-weather/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  fetchFromGitHub,
+  gcc,
+  gnumake,
+  lib,
+  ncurses,
+  stdenv,
+}:
+stdenv.mkDerivation {
+  name = "ascii-weather";
+  version = "1.0.0";
+  src = fetchFromGitHub {
+    owner = "NewDawn0";
+    repo = "asciiWeather";
+    rev = "70bf111647d064c3fcd0fe672b9fa697f4d060e4";
+    hash = "sha256-Dcosx6iEnvFCMrmUS7gSLg8re5zl1BXWX/Nu6hr4Pgw=";
+  };
+  buildInputs = [
+    gcc
+    gnumake
```

Comment:
stdenv already comes with them
```suggestion
```

Reply:
When I remove gcc and gnumake It does not build as gcc is not found. I don't know if this is only specifc to the stdenv on mac, which is why I'll leave it in for now.

Reply:
Ah, stdenv uses clang by default on Darwin. Use gccStdenv if you specifically need gcc